rac-gcp-deploy/orb.yml

```diff
@@ -288,6 +281,88 @@ jobs:
           command: sbt service-test:test
           no_output_timeout: 20m
 
+  snyk_test:
+    parameters:
+      executor:
+        description: "Name of executor to use for this job. Defaults to machine executor"
+        type: executor
+        default: docker
+      workspace-dir:
+        description: "Path to restore the workspace"
+        type: string
+        default: ~/project
+      cache-suffix:
+        description: "Suffix to use for the cache key, e.g. -v2, to be able to clear the cache"
+        default: ""
+        type: string
+    executor: <<parameters.executor>>
+    steps:
+      - attach_workspace:
+          at: <<parameters.workspace-dir>>
+      - load_cache:
+          cache-suffix: <<parameters.cache-suffix>>
+      - run: *sbt-dependency-graph
+      - snyk/scan:
+          project: $CIRCLE_PROJECT_REPONAME
+          fail-on-issues: true
```

Comment:
Will there be a way to override the fail-on-issues if we get stuck with a particular dependency?
(It's a bit of a risk though as you'd be allowing broken windows...)

Reply:
At the moment there isn't any override.